src/models/style-model/__tests__/index.spec.js

```diff
@@ -72,7 +84,67 @@ describe('createStyleModel', () => {
   });
 
   it('query should have correct properties', () => {
-    expect(create().query).to.have.all.keys(['axis', 'label', 'referenceLine']);
+    expect(create().query).to.have.all.keys(['axis', 'label', 'referenceLine', 'legend']);
+  });
+
+  it('query should have only four keys', () => {
+    expect(Object.keys(create().query).length).to.equal(4);
+  });
```

Comment:
This test feels like a duplication of the one above :)

Reply:
I want to include it to ensure that if someone adds additional styling to the style-model this will fail and force them to at least correct it but hopefully they'll also add tests for whatever styling they're adding.

Reply:
`to.have.all.keys` will fail if you add more keys.

Reply:
Didn't think it would but i'll remove the length check then as it doesnt fill a purpose.